AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2196,6 +2247,14 @@ - (BOOL)shouldBypassEnsureDisplay
 #pragma mark - For Subclasses
 
 - (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
```

Comment:
`calculateLayoutThatFits:` and related methods in `ASLayoutable.h` are declared within a  `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`. 

We don’t have any nullability declarations in `ASDisplayNodeInternals.h` where _calculatedDisplayNodeLayout declared is. I filed a ship task for that as we should do this in a separate PR.